src/panels/profile/ha-refresh-tokens-card.ts

```diff
@@ -35,6 +35,18 @@ class HaRefreshTokens extends LitElement {
       refreshTokens?.filter((token) => token.type === "normal").reverse()
   );
 
+  compareTokenLastUsedAt = (tokenA: RefreshToken, tokenB: RefreshToken) => {
+      const timeA = new Date(tokenA.last_used_at);
+      const timeB =  new Date(tokenB.last_used_at);
+      let comparison = 0;
+      if (timeA < timeB) {
+        comparison = 1;
+      } else if (timeA > timeB) {
+        comparison = -1;
+      }
+      return comparison;
```

Comment:
We should return as soon as possible
```suggestion
      if (timeA < timeB) {
        return 1;
      } else if (timeA > timeB) {
        return -1;
      }
      return 0;
```

Reply:
To solve the `lint` issue, would I need to change the `new Date()` overload, or break it out and do `undefined` checks before comparing with the values?

Reply:
yes, you should check for undefined first, as:
```
new Date(undefined) -> Invalid Date
```

Reply:
Ok. I made a change in my last commit that should resolve that, but it looks like you might be suggesting a syntax I'm not familiar with.